presentation/src/main/java/com/going/presentation/entertrip/createtrip/choosedate/CreateTripActivity.kt

```diff
@@ -153,17 +151,16 @@ class CreateTripActivity :
 
     private fun initNextBtnClickListener() {
         binding.btnCreateTripNext.setOnSingleClickListener {
-            Intent(this, EnterPreferenceActivity::class.java).apply {
-                putExtra(NAME, viewModel.name.value)
-                putExtra(START_YEAR, viewModel.startYear.value)
-                putExtra(START_MONTH, viewModel.startMonth.value)
-                putExtra(START_DAY, viewModel.startDay.value)
-                putExtra(END_YEAR, viewModel.endYear.value)
-                putExtra(END_MONTH, viewModel.endMonth.value)
-                putExtra(END_DAY, viewModel.endDay.value)
-                addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
-                startActivity(this)
-            }
+            EnterPreferenceActivity.createIntent(
+                this,
+                viewModel.name.value ?: "",
```

Comment:
이전에 상호쌤이 썼던 것 처럼 아래처럼 하면 좀 더 깔끔해용~ 보기조아집미다

```suggestion
                this,
                viewModel.name.value.orEmpty(),
```

Reply:
오!! 바로 바꾸겠습니다 감사합니다아 :)